admin/app/view_models/workarea/admin/pricing_sku_view_model.rb

```diff
@@ -62,7 +62,9 @@ def on_sale?
       #
       # @return [String]
       def sell_price
+        return 0.to_m.format if min_price.blank?
```

Comment:
Selling it for `$0` is meaningfully different than not having a price specified. Let's return `nil` instead and handle it with a en dash or something in display to indicate it's blank.